src/main/java/org/jenkins/plugins/lockableresources/LockableResource.java

```diff
@@ -36,12 +36,12 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.Set;
 import java.util.concurrent.ExecutionException;
 import java.util.logging.Level;
 import java.util.logging.Logger;
 import jenkins.model.Jenkins;
-import org.apache.commons.lang3.StringUtils;
```

Comment:
This only worked because common lang 3 was provided as transitive dependency via data-tables-api plugin